python/lsst/afw/cameraGeom/cameraSys.py

```diff
@@ -21,7 +21,7 @@
 #
 from __future__ import absolute_import, division, print_function
 
-from lsst.afw.geom import addTransformMapMethods
+from lsst.afw.geom.python import addTransformMapMethods
```

Comment:
The previous commit added a new function `lsst.afw.geom.python.addTransformMethods` but left the pre-existing function `lsst.afw.geom.addTransformMapMethods` where it was. This commit moves that second function down into the `python` sub-namespace.